src/main/webapp/scripts/tests/analyze-test.js

```diff
@@ -17,8 +17,9 @@ QUnit.testStart(() => {
 });
 
 QUnit.test('DOM updates with sentiment score', function (assert) {
+  console.log('sentiment score');
   const body = { 'score': SCORE, 'scoreAvailable': true };
-  const response = { json: () => { return body }, status: 200 };
+  const response = { json: () => { return body }, status: 200, ok: true };
```

Comment:
Nit-ish: I believe if you make a response object it would imitate closer to what fetch returns

https://stackoverflow.com/a/44044858
https://developer.mozilla.org/en-US/docs/Web/API/Response/Response